frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.tsx

```diff
@@ -141,28 +146,40 @@ export const EmbedFrame = ({
   const hasVisibleParameters = visibleParameters.length > 0;
 
   const hasHeader = Boolean(finalName || dashboardTabs) || downloadsEnabled;
-  const canParameterPanelSticky =
+
+  const allowParameterPanelSticky =
     !!dashboard && isParametersWidgetContainersSticky(visibleParameters.length);
+  const shouldApplyParameterPanelThemeChangeTransition =
+    !isParameterPanelStickyStateChanging && isParameterPanelSticky;
 
   return (
     <Root
       hasScroll={hasFrameScroll}
       isBordered={bordered}
       hasVisibleOverflowWhenPriting={isPublicDashboard}
-      className={cx(EmbedFrameS.EmbedFrame, className, {
-        [EmbedFrameS.NoBackground]: !background,
-      })}
+      className={cx(
+        className,
+        EmbedFrameS.EmbedFrame,
+        TransitionS.transitionThemeChange,
```

Comment:
@metabase/embedding what our preference regarding styling is?
- like here and below - add the `TransitionS.transitionThemeChange` to each specific element?
- or like we do it in CSS Module add it from there from a single place using selectors (see example below)

Example:
```
  .EmbedFrame .ConentContainer,
  .EmbedFrame .EmbedFrameHeader,
  .EmbedFrame .EmbedFrameSeparator,
  .EmbedFrame .EmbedFrameParametersWidgetContainer,
  .EmbedFrame .EmbedFrameFooter {
    transition: var(--transition-theme-change);
  }
```

Reply:
My personal preference would be 2) as we can see in one place which components the theme change transitions are being applied to.